robolectric-resources/src/main/java/org/robolectric/res/ResType.java

```diff
@@ -49,6 +57,27 @@ private static TypedResource extractScalarItems(XpathResourceXmlLoader.XmlNode x
     return new TypedResource<>(typedResources, arrayResType, xmlContext);
   }
 
+  private static TypedResource extractTypedItems(XpathResourceXmlLoader.XmlNode xmlNode, ResType arrayResType, XmlLoader.XmlContext xmlContext) {
+    final List<TypedResource> items = new ArrayList<>();
+    for (XpathResourceXmlLoader.XmlNode item : xmlNode.selectElements("item")) {
+      final String itemString = item.getTextContent();
+      ResType itemResType = inferFromValue(itemString);
+      if (itemResType == ResType.CHAR_SEQUENCE) {
+        if (itemString.startsWith("?")) {
```

Comment:
Can we use AttributeResource.isStyleReference(String value) here? Feel free to refactor to something like ResourceValue - probably makes sense since its not always associated with an AttributeResource